storybook/.storybook-html/webpack.config.js

```diff
@@ -11,6 +11,21 @@ module.exports = ({ config, mode }) => {
   // You can change the configuration based on that.
   // 'PRODUCTION' is used when building the static version of storybook.
 
+  config.resolve.extensions.push('.svg');
+
+  config.module.rules = config.module.rules.map( data => {
+      if (/svg\|/.test( String( data.test ) ))
+          data.test = /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|cur|ani)(\?.*)?$/;
```

Comment:
I don't understand what this is testing for. Is the module rule for svgs not just that the filename extension is `.svg`?

Reply:
you're right.  As with all webpack dealings, this is a result of me copying over a bunch of webpack configurations to see if they work - this is one that doesn't work, but isn't visible because storybook-twig doesn't show icons anyway.  Better to take it out.